.github/workflows/commenter.yml

```diff
@@ -0,0 +1,34 @@
+name: Make Change Comment
+
+on:
+  workflow_run:
+    workflows: ["Consitency"]
```

Comment:
```suggestion
    workflows: ["Consistency"]
```

Reply:
Not sure if we need to change this name somewhere else as well

Reply:
nope, it is spelled correctly in consitency.yml, just a typo